banano-quest/banano-quest/Models/Winners+CoreDataClass.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Winners+CoreDataClass.swift
+//  banano-quest
+//
+//  Created by Pabel Nunez Landestoy on 6/26/18.
+//  Copyright © 2018 Michael O'Rourke. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(Winners)
+public class Winners: NSManagedObject {
+    
+    convenience init(obj: [AnyHashable: Any]!, context moc: NSManagedObjectContext) {
+        self.init(context: moc)
+        self.address = obj["address"] as? String
+    }
+    
+    func save() {
+        do {
+            try self.managedObjectContext?.save()
+        } catch let error as NSError {
```

Comment:
Remove the catch, add throws.

Reply:
I need to catch the error since is inside a handler.